src/Keboola/DbWriter/Writer/Snowflake.php

```diff
@@ -410,6 +408,11 @@ public function generateStageName($runId = null)
         );
     }
 
+    public function getCurrentUser()
+    {
+        return $this->db->fetchAll("SELECT CURRENT_USER;")[0]['CURRENT_USER'];
```

Comment:
I get worried when I see `[0]` assumption.  Is it guaranteed that there will be a result?